### PR TITLE
fix: Rename authenticators to security methods

### PR DESCRIPTION
**Pull request by jmelberg-okta**
### :warning: Important :warning:
This PR is to be included in the release scheduled the week of July 12th, 2021. It should not be merged into `master` or a release branch until then.

Renames instances where more than one authenticator is used to "security methods".

### Reviewer

- @mauriciocastillosilva-okta 

### Issue:

- [OKTA-405261](https://oktainc.atlassian.net/browse/OKTA-405261)

### CODE REVIEW

**Review by mauriciocastillosilva-okta — APPROVED** [review]

File: packages/@okta/i18n/src/properties/login_da.properties [hunk 1]
```diff
@@ -202,6 +202,20 @@ model.validation.field.invalid.format.ipv4 = Denne v\u00E6rdi er ikke en gyldig
 model.validation.field.invalid.format.hostname = Denne v\u00E6rdi er ikke et gyldigt v\u00E6rtsnavn
 model.validation.field.username = Kontroll\u00E9r dit brugernavn
 
+schema.validation.field.value.must.string = V\u00E6rdien skal v\u00E6re en streng
```
Comment (jmelberg-okta) [inline packages/@okta/i18n/src/properties/login_da.properties:205]: There is additional strings included from [this commit into `master`](https://github.com/okta/okta-signin-widget/commit/8926c906e987e752fffd4300ccd62ef631cfd021). These are English leaks that should be resolved ASAP.

@nikhilvenkatraman-okta - can you confirm this?

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "update-readme": "node scripts/buildtools update-readme",
     "retirejs": "retire --jspath target/js --package",
     "lint:eslint": "eslint .",
-    "lint:eslint:report": "eslint -f checkstyle -o build2/OSW-eslint-checkstyle-result.xml .",
+    "lint:eslint:report": "eslint -f checkstyle -o build2/OSW-eslint-checkstyle-result.xml . --quiet",
```
Comment (jmelberg-okta) [inline package.json:29]: Required now that we're reporting warnings for inclusive language (`.*master.*`) due to [this change](https://github.com/okta/okta-signin-widget/blob/master/packages/%40okta/i18n/src/properties/login.properties#L210).

Comment (nikhilvenkatraman-okta) [inline package.json:29]: why do we need this change ?

**Review by nikhilvenkatraman-okta — APPROVED** [review]

File: packages/@okta/i18n/src/properties/login_da.properties (hunk 1 above)
Comment (nikhilvenkatraman-okta) [reply packages/@okta/i18n/src/properties/login_da.properties:205]: resolved offline

**Pull request closed without merging**